<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductDialog</class>
 <widget class="QDialog" name="AddProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Product</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Add New Product</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">
       font-size: 22px;
       font-weight: bold;
       padding: 12px;
       color: #eaeef1ff;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formWidget">
     <layout class="QFormLayout" name="formLayout">
      
      <!-- Product ID (read-only) -->
      <item row="0" column="0">
       <widget class="QLabel" name="labelProductId">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="productIdEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: #ecf0f1;
          color: #34495e;
          font-weight: bold;
          padding: 4px 6px;
          border: 1px solid #bdc3c7;
          border-radius: 4px;
         </string>
        </property>
       </widget>
      </item>

      <!-- Product Name -->
      <item row="1" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Product Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="productNameEdit">
        <property name="styleSheet">
         <string notr="true">
          padding: 4px 6px;
          border: 1px solid #bdc3c7;
          border-radius: 4px;
         </string>
        </property>
       </widget>
      </item>

      <!-- Category -->
      <item row="2" column="0">
       <widget class="QLabel" name="labelCategory">
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="categoryComboBox">
        <item>
         <property name="text">
          <string>Electronics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apparel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Groceries</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Books</string>
         </property>
        </item>
       </widget>
      </item>

      <!-- Quantity -->
      <item row="3" column="0">
       <widget class="QLabel" name="labelQuantity">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="quantitySpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="styleSheet">
         <string notr="true">
          padding: 2px 6px;
         </string>
        </property>
       </widget>
      </item>

      <!-- Price -->
      <item row="4" column="0">
       <widget class="QLabel" name="labelPrice">
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="priceSpinBox">
        <property name="minimum">
         <double>0.00</double>
        </property>
        <property name="maximum">
         <double>1000000.00</double>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="singleStep">
         <double>1.00</double>
        </property>
        <property name="styleSheet">
         <string notr="true">
          padding: 2px 6px;
         </string>
        </property>
       </widget>
      </item>

      <!-- Description -->
      <item row="5" column="0" colspan="1">
       <widget class="QLabel" name="labelDescription">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="1">
       <widget class="QTextEdit" name="descriptionTextEdit">
        <property name="styleSheet">
         <string notr="true">
          border: 1px solid #bdc3c7;
          border-radius: 4px;
          padding: 6px;
          font-size: 14px;
         </string>
        </property>
        <property name="minimumHeight">
         <number>80</number>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonWidget">
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: #27ae60;
          color: white;
          padding: 10px 20px;
          border-radius: 8px;
          font-weight: bold;
          font-size: 16px;
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
          background-color: #c0392b;
          color: white;
          padding: 10px 20px;
          border-radius: 8px;
          font-weight: bold;
          font-size: 16px;
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
